python_embed.c

```diff
@@ -117,6 +117,7 @@ static PyStructSequence_Field player_state_fields[] = {
     {"powerups", "The player's powerups."},
     {"holdable", "The player's holdable item."},
     {"flight", "A struct sequence with flight parameters."},
+    {"is_chatting", "Whether the player is currently chatting."},
     {"is_frozen", "Whether the player is frozen(freezetag)."},
```

Comment:
After you apply my suggestion from https://github.com/MinoMino/minqlx/pull/121/files#r1292673077, this also needs to be changed.